chain/stmgr/stmgr.go

```diff
@@ -1120,3 +1123,11 @@ func (sm *StateManager) GetCirculatingSupply(ctx context.Context, height abi.Cha
 
 	return csi.FilCirculating, nil
 }
+
+func (sm *StateManager) GetNtwkVersion(ctx context.Context, height abi.ChainEpoch) runtime.NetworkVersion {
+	if build.UpgradeBreezeHeight == 0 || height <= build.UpgradeBreezeHeight {
```

Comment:
Cool name.  Is the next one going to be hurricane?

Reply:
Confirming this means that `UpgradeBreezeHeight` will still run protocol version 0.  `UpgradeBreezeHeight + 1` is version 1